instrumentation/log4j/log4j-appender-2.17/library/README.md

```diff
@@ -20,7 +20,6 @@ For Maven, add to your `pom.xml` dependencies:
     <groupId>io.opentelemetry.instrumentation</groupId>
     <artifactId>opentelemetry-log4j-appender-2.17</artifactId>
     <version>OPENTELEMETRY_VERSION</version>
-    <scope>runtime</scope>
```

Comment:
good catch, thanks!

can you change the gradle configuration below also?

```
runtimeOnly("io.opentelemetry.instrumentation:opentelemetry-log4j-appender-2.17:OPENTELEMETRY_VERSION")
```

to

```
implementation("io.opentelemetry.instrumentation:opentelemetry-log4j-appender-2.17:OPENTELEMETRY_VERSION")
```

Reply:
I knew that I saw "runtime" in one more place in the doc. Thanks for catching gradle one, too!